tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -2982,6 +2982,12 @@ def plot_ensemble_model(self, prune_unused_nodes=True) -> str:
         best_model_node = self.get_model_best()
 
         A = nx.nx_agraph.to_agraph(G)
+        
+        for node in A.iternodes():
+            # Remove unnecessary attributes
+            label = f"{node.name}\nVal score: {float(node.attr['val_score']):.4f}"
+            node.attr.clear()
+            node.attr['label'] = label
```

Comment:
Wouldn't this crash? Don't you need to keep `node.name`? Also, don't you need to remove L2997 which also sets `label` attribute using the currently deleted attributes?

Reply:
`node.name` is not part of the attr I believe. And L2997 should be removed, that was a mistake.